jpg323(ShadowsOfTheKnight)/gameCode.py

```diff
@@ -0,0 +1,82 @@
+# ***************************
+# Shadows of the Night - Ep1
+# ***************************
+#
+# The purpose of this program is to solve the puzzle: `SHADOWS OF THE KNIGHT - EPISODE 1 <https://www.codingame.com/training/medium/shadows-of-the-knight-episode-1>`_,
+# from the educational website `codingames.com <https://www.codingame.com/home>`_.
+#
+# In the game there is a grid of windows for an apartment building and one of those windows has a bomb in it. The game ends when the user arrives at the bomb. The user can jump to any window in the grid by changing their x and y coordinates, but are only given the direction in which the bomb is in relation to their current position.
+#
+# Goal
+# ====
+# The objective of the game is to get to the bomb in as few jumps as possible.
+#
+# You are given the size of the grid, the starting location and direction of the bomb. The number of jumps you are allowed to make is limited and the game will provide the direction in which the target is.
+#
+#
+# Inputs
+# ======
+# Below are variables given by the game and are different for various test cases.
+#
+#   * **w**: width of the building. Provided only once. (int)
+#   * **h**: height of the building. Provided only once. (int)
+#   * **n**: maximum number of turns the user has to make it to the bomb. Provided only once. (int)
+#   * **bomb_dir**: The direction of the bombs from batman's current location (U, UR, R, DR, D, DL, L or UL). Provided each round. The letters stand for Up, Down, Left, and Right. However, here it is just initialized as "AB" for organization. (string)
+# 
+import sys
+import math
+w, h = [int(i) for i in input().split()]
+n = int(input())  # maximum number of turns before game over.
+x0, y0 = [int(i) for i in input().split()]
+bomb_dir = "AB"
+# Approach
+# ========
+# | The approach we are taking is based on a binary search tree. We will check what *directions* are in the **bomb_dir** string and write if-statements for each case.
```

Comment:
Don't use a line block (`# | The approach...`); instead, use `# The approach...`.